### DISCUSSION TITLE
File uploaded as public in drive .

### DISCUSSION THREAD

**Discussion by sp003** [anasty17/mirror-leech-telegram-bot#1122] [Announcements]
The Bot is uploading every file as public in Google drive .

This leads to rate limiting very quickly .

This also leads to flagging some files by Google . So files are getting locked .

How can I force bot to upload files as restricted ?
(only accessible by teamdrive members )

I tried changing the bot folder share settings to restricted 

but still files are uploaded as public & everyone can download without joining my Google group .

**Comment by anasty17** [comment]
IS_TEAM_DRIVE =true
